octoprint_mrbeam/__init__.py

```diff
@@ -683,6 +696,8 @@ def on_settings_save(self, data):
             octoprint.plugin.SettingsPlugin.on_settings_save(self, data)
 
     def set_heavy_duty_prefilter(self, value):
+        if value != self._settings.get(["heavyDutyPrefilter"]):
+            self._send_maintenance_information(trigger="heavyDutyPrefilter_changed")
```

Comment:
Maybe it makes sense to group the different triggers in an enum or similar?

Reply:
ahh sure, I added them for debugging but it makes sense to keep them and change them to enums or constants